test/jdk/javax/swing/JButton/bug4490179.java

```diff
@@ -81,10 +82,11 @@ public static void main(String[] args) throws Exception {
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             robot.mousePress(InputEvent.BUTTON3_DOWN_MASK);
             robot.mouseRelease(InputEvent.BUTTON3_DOWN_MASK);
+            robot.delay(3000);
+            boolean result = passed;
             robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
-            robot.delay(500);
 
-            if (!passed) {
+            if (!result) {
```

Comment:
Perhaps, we should ensure that an `ActionEvent` gets delivered at this point. Is it expected or not?